registry/storage/filewriter.go

```diff
@@ -72,7 +72,7 @@ func newFileWriter(ctx context.Context, driver storagedriver.StorageDriver, path
 	buffered := bufferedFileWriter{
```

Comment:
@aibaars I've made an example, available in the comment below, that shows this is actually unnecessary. The new test case might be bad.